data/thebarshablog.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Barsha Das",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Founder of **The CookieButton Project** | Co-organizer of **KubeHuddle Toronto 2023** | **Women In Web3 2022** by CoinDCX",
+  "tags": ["Open Source", "DevOps", "Web Development","Cloud Computing", "Platform Engineering", "Network Engineering"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/thebarshablog" },
+    { "icon": "FaGithub", "url": "https://github.com/thebarshablog" },
+    { "icon": "website", "url": "https://thebarshablog.github.io" }
+  ],
+  "avatar": "https://github.com/thebarshablog.png",
+  "links": [
+    {
+      "name": "Daily Tweets on Tech & Life",
+      "url": "https://twitter.com/thebarshablog",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Let's Contribute Together",
+      "url": "https://github.com/thebarshablog",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Track My Career Updates",
+      "url": "https://www.linkedin.com/in/thebarshablog/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Read Me",
+      "url": "https://thebarshablog.github.io",
+      "icon": "FaGlobe"
+    },
+    {
+      "name": "Support My Initiatives",
+      "url": "https://www.buymeacoffee.com/thebarshablog",
+      "icon": "FaDollarSign"
+    }
+  ],
+  "Let's Talk": [
+    {
+      "title": "Speak To Me 1:1 for FREE",
+      "icon": "SiKakaotalk",
+      "color": "grey",
+      "description": "Speak to me about yourself",
+      "url": "https://calendly.com/thebarshablog"
+    }
+  ],
+  "Events": [
```

Comment:
Each event has it's own file in your events folder, here are the docs https://linkfree.eddiehub.io/docs/how-to-guides/events

Any questions or improvements to the docs let us know

Reply:
Implemented the steps mentioned in the docs.

Reply:
Ok great, you can now remove this section in here because you have the file data/thebarshablog/events/2023-05-16-kubehuddle-conference.json

Or is this meant to be `milestones` instead of `Events`?  https://linkfree.eddiehub.io/docs/how-to-guides/milestones